data/tlg0007/tlg071/tlg0007.tlg071.perseus-eng4.xml

```diff
@@ -0,0 +1,1125 @@
+         		<name>Lisa Cerrato</name>
+         		<name>Rashmi Singhal</name> 
+         		<name>Bridget Almas</name>       		
+         	</respStmt>
+         	<funder n="org:NEH">The National Endowment for the Humanities</funder>
+         </titleStmt>
+      	<publicationStmt>
+      	<publisher>Trustees of Tufts University</publisher>
+      	<pubPlace>Medford, MA</pubPlace>
+      	<authority>Perseus Project</authority>
+      	<date type="release">2010-12-13</date>
+      	</publicationStmt>
+      	<sourceDesc>
+            <biblStruct>
+               <monogr>
+                  <author>Plutarch</author>
+                  <title>Plutarch's Morals.</title>
+                  <respStmt>
+                     <resp>Translated from the Greek by several hands. Corrected and revised by</resp>
+                     <name>William W. Goodwin, PH. D.</name>
+                  </respStmt>
+                  <imprint>
+                     <pubPlace>Boston</pubPlace>
+                     <publisher>Little, Brown, and Company</publisher>
+                     <pubPlace>Cambridge</pubPlace>
+                     <publisher>Press Of John Wilson and son</publisher>
+                     <date>1874</date>
+                  </imprint>
+                  <biblScope unit="volume">2</biblScope>
+               </monogr>
+            	<ref target="https://archive.org/details/plutarchsmoralst02plutuoft/page/446">The Internet Archive</ref>
+            </biblStruct>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+      	<p>Text encoded in accordance with the latest EpiDoc standards</p> 
+      	<p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE Architecture</p> 
+         <refsDecl n="CTS">
+         	<cRefPattern n="section" matchPattern="(\w+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">
+         		<p>This pointer pattern extracts sections</p></cRefPattern>
+            <!--<state unit="section"/>-->
+         </refsDecl>
+      	<!--<refsDecl doctype="TEI.2">
+				<state unit="stephpage" n="chunk"/>
+				</refsDecl>-->
+      </encodingDesc>
+      <profileDesc>
+      	<langUsage>
+      		<language ident="eng">English</language>
+      		<language ident="grc">Greek</language>
+      		<language ident="lat">Latin</language>
+      	</langUsage>
+      </profileDesc>
+      <revisionDesc>
+      	<change who="GRC" when="2006">tagging</change>
+      	<change who="Sophia Elzie" when="2019-06-21">CTS and Epidoc conversion</change>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="eng">
+      <body>
+      	<div type="translation" xml:lang="eng" n="urn:cts:greekLit:tlg0007.tlg071.perseus-eng4">
+         <head>How a man may be sensible of his progress in virtue.</head>
+         <pb xml:id="v.2.p.446"/>
+          <div type="textpart" subtype="section" n="1">
+            <head>My friend Sossius Senecio,</head>
```

Comment:
hmm, this is more of a salutation but since this isn't a letter, EpiDoc won't let us change this.

How about? `<head type="salute">My friend Sossius Senecio,</head>`